ci/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+RUN ["apt-get", "update"]
+RUN ["apt-get", "install", "-y", "gcc"]
+RUN ["apt-get", "install", "-y", "g++"]
+RUN ["apt-get", "install", "-y", "cmake"]
+
+WORKDIR /build/
+COPY ./ ./project
+
+VOLUME /build/tools
+ENV TOOLS_DIR "/build/tools"
+
+VOLUME /build/auth
+ENV AUTH_DIR "/build/auth"
+
+# Add the spatial wrapper script to the PATH.
```

Comment:
Where is spatial coming from? Since you're downloading it dynamically, you shouldn't couple the container to the download location of the `spatial` binary.

Reply:
This is the `spatial` wrapper script in the `ci` folder.